.github/workflows/release.yml

```diff
@@ -26,14 +26,14 @@ jobs:
       id: info
       run: |
         if [ '${{ github.event_name }}' = 'pull_request' ]; then
-          echo '::set-output name=commit_message::'
+          echo "commit_message=" >> $GITHUB_ENV
```

Comment:
This and all the others are actually `echo 'name=commit_message' >> $GITHUB_OUTPUT`
reference: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/